ksqldb-engine/src/main/java/io/confluent/ksql/util/BinPackedPersistentQueryMetadataImpl.java

```diff
@@ -234,11 +235,6 @@ public void stop() {
                                                 + " individual queries in shared runtime");
   }
 
-  @Override
```

Comment:
Was this just unused? We may want to look into why if so. that might affect some functionality

Reply:
I'll investigate quickly to make sure, but the fact that it was also unused in `QueryMetadataImpl` leads me to believe it was just an unused getter for a field that is only needed within the class (which jives with my understanding of what the `materializationProvider` is/does)